documentation/README.md

```diff
@@ -28,7 +28,7 @@ Here are each of the documenation pages that are generated, the files
 that you may need to modify to make changes to the generated documentation,
 and the rake task used to generate the documentation.
 
-### Bolt command reference
+### Shell command reference
```

Comment:
```suggestion
### *nix command reference
```

Or something? Powershell is also a shell :P

Reply:
Tbf, 'Shell command' isn't inaccurate since they can still be used in PowerShell. :P Probably best to have consistency with the command snippets in the docs, though.